### PR TITLE
Reloop Beatmix gen.1 documentation page added.

### PR DESCRIPTION
**Pull request by uncleeugene**
Reloop Beatmix gen.1 description page added.

https://github.com/mixxxdj/mixxx/pull/12422

https://deploy-preview-598--mixxx-manual.netlify.app/hardware/controllers/reloop_beatmix

### CODE REVIEW

File: source/hardware/controllers/reloop_beatmix.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+-  `Forum thread <https://mixxx.discourse.group/t/reloop-beatmix-1st-gen-mapping-implemented/28536>`__
+
+.. versionadded:: 2.4
+
+Audio
+-----
+
+This controller is a class complaint USB audio device. :term:`ASIO` driver for Windows based setups provided by manufacturer.
+
+.. figure:: ../../_static/controllers/reloop_beatmix.png
```
Comment (ronso0) [inline source/hardware/controllers/reloop_beatmix.rst:20]: Do you mind putting bigger button/knob labels on top?
They're pretty hard to read in a regukar browser view.

**Comment by ronso0** [comment]
Thank you for this, and the mapping of course.
Just took a quick look, will look closer later on.

**Comment by ronso0** [comment]
Please check the CI.
pre-commit also check for typos
```
source/hardware/controllers/reloop_beatmix.rst:42: explainatory ==> explanatory
source/hardware/controllers/reloop_beatmix.rst:52: explainatory ==> explanatory
```

File: source/hardware/controllers/reloop_beatmix.rst (hunk 1 above)
Comment (uncleeugene) [reply source/hardware/controllers/reloop_beatmix.rst:20]: Ok, i will update the pic. It will require some work though. This current one is just a rip from user manual, i gonna need to make a new one myself.

Comment (uncleeugene) [reply source/hardware/controllers/reloop_beatmix.rst:20]: I mean it'll take some time to get it done :)

Comment (ronso0) [reply source/hardware/controllers/reloop_beatmix.rst:20]: I just thought about putting some slightly larger text bubbles on top of it with Gimp or alike, no need to create an entirely new image.
If it's too much effort, leave it as it is.

File: source/hardware/controllers/reloop_beatmix.rst [hunk 2]
```diff
@@ -17,63 +17,74 @@ Audio
...
+* 26 - Booth/rec output. Channels 3-4. These are also routed to internal headphone amp.
+* 27 - USB port.
 
 Front
 -----
-* F34 - Mic input. This might be turned off, routed to outputs or to a software.
-* F35 - Mic gain potentiometer.
-* F36 - Mic routing switch.
-* F37 - Headphone output.
+.. figure:: ../../_static/controller/reloop_beatmix_front.png
```
Comment (ronso0) [inline source/hardware/controllers/reloop_beatmix.rst:76]:
```suggestion
.. figure:: ../../_static/controllers/reloop_beatmix_front.png
```

File: source/hardware/controllers/reloop_beatmix.rst [hunk 3]
```diff
@@ -17,63 +17,74 @@ Audio
...
 
 Back
 ----
-* B29 - Kensington lock.
-* B30 - Jog sensitivity potentiometers.
-* B31 - Master output. Channels 1-2 on USB audio device.
-* B32 - Booth/rec output. Channels 3-4. These are also routed to internal headphone amp.
-* B33 - USB port.
+
+.. figure:: ../../_static/controller/reloop_beatmix_back.png
```
Comment (ronso0) [inline source/hardware/controllers/reloop_beatmix.rst:62]:
```suggestion
.. figure:: ../../_static/controllers/reloop_beatmix_back.png
```

File: source/hardware/controllers/reloop_beatmix.rst [hunk 4]
```diff
@@ -17,63 +17,74 @@ Audio
 
 This controller is a class complaint USB audio device. :term:`ASIO` driver for Windows based setups provided by manufacturer.
 
-.. figure:: ../../_static/controllers/reloop_beatmix.png
+Deck Controls
+-------------
+
+.. figure:: ../../_static/controller/reloop_beatmix_top.png
```
Comment (ronso0) [inline source/hardware/controllers/reloop_beatmix.rst:23]:
```suggestion
.. figure:: ../../_static/controllers/reloop_beatmix_top.png
```

**Review by ronso0 — CHANGES_REQUESTED** [review]
the image paths are missing an `s`

File: source/hardware/controllers/reloop_beatmix.rst (hunk 2 above)
Comment (uncleeugene) [reply source/hardware/controllers/reloop_beatmix.rst:76]: <facepalm/> :) Sorry for this..

File: source/hardware/controllers/reloop_beatmix.rst [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+* 13 - Momentary **pitch up/down** buttons. Self explanatory i think. **Shift**-buttons will fast forward and rewind.
+* 14 - **SHIFT** button is used to access all of second layer controls. Worth noticing, there is one SHIFT button for each side which affects its own side only. Except for a navigation dial. Both shift buttons deal the same way with the navigation dial.
+* 15 - **HOTCUES**. Three hotcues are set/activated by pressing corresponding buttons and cleared by **Shift**-button.
+* 16 - **SYNC**. Toggles sync mode of the corresponding deck. **Shift**-button will reverse the playback.
+* 17 - **CUE PLAY**. Button set cue and start playing on release. **Shift**-button acts as a tap tempo.
+* 18 - **CUE**. **Shift**-button toggles Fader Start feature.
+* 19 - **PLAY/PAUSE**. **Shift**-button toggles keylock mode.
+* 20 - **EQ and MASTER SECTION** this section is self explanatory, just one thing worth noticing: master output control is pure hardware, it doesn't talk to software at all.
+* 21 - **LOAD**. Loads a track into a corresponding deck. Button is only lit when deck is stopped and is safe to load. **Shift**-button toggles scratch mode as it was described above.
```
Comment (ronso0) [inline source/hardware/controllers/reloop_beatmix.rst:51]: > Button is only lit when deck is stopped and is safe to load.

Is this still the case? I can't spot where this is controlled in the manual.
Also, I think "safe" is up to the users here, they can decide whether tracks can be loaded into playing decks, and if yes, if the deck keeps playing, see Preferences -> Decks.

Futrhermore, LED off doesn't mean the button is inactive / unresponsive, and after a while with a controller you don't actually look at browse/load controls anymore while operating due to muscle memory (and dial/load are so close)

Also LED ON usually means something IS loaded. (samplers, effec ON)

Long story short: I think it would be better to bind the LED to "track loaded".
And make Shift+Load do eject as mentioned in the mapping PR

File: source/hardware/controllers/reloop_beatmix.rst [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+* 11 - **SEARCH**. Currently this button increase the jog sensitivity by 4 which makes it more suitable for searching through a loaded track. It also affects pitch control function of the jog, but not the scratch function.
+* 12 - **Jog**. It is a two zone control wheel. Top plate is touch sensitive and it will cause a deck to scratch when touched if scratch mode is on. Rim is scratch-indifferent.
+* 13 - Momentary **pitch up/down** buttons. Self explanatory i think. **Shift**-buttons will fast forward and rewind.
+* 14 - **SHIFT** button is used to access all of second layer controls. Worth noticing, there is one SHIFT button for each side which affects its own side only. Except for a navigation dial. Both shift buttons deal the same way with the navigation dial.
+* 15 - **HOTCUES**. Three hotcues are set/activated by pressing corresponding buttons and cleared by **Shift**-button.
+* 16 - **SYNC**. Toggles sync mode of the corresponding deck. **Shift**-button will reverse the playback.
+* 17 - **CUE PLAY**. Button set cue and start playing on release. **Shift**-button acts as a tap tempo.
```
Comment (ronso0) [inline source/hardware/controllers/reloop_beatmix.rst:47]: there is no tempo tap (before https://github.com/mixxxdj/mixxx/pull/12104 is merged ; )
You mean BPM tap https://manual.mixxx.org/2.4/en/chapters/appendix/mixxx_controls.html#control-[ChannelN]-bpm_tap
